test/test_prototype_transforms.py

```diff
@@ -152,3 +153,56 @@ def test_normalize(self, transform, input):
     )
     def test_random_resized_crop(self, transform, input):
         transform(input)
+
+
+class TestRandomHorizontalFlip:
+    def input_tensor(self, dtype=torch.float32):
+        return torch.tensor([[[0, 1], [0, 1]], [[1, 0], [1, 0]]], dtype=dtype)
+
+    def expected_tensor(self, dtype=torch.float32):
+        return torch.tensor([[[1, 0], [1, 0]], [[0, 1], [0, 1]]], dtype=dtype)
```

Comment:
1. The names are somewhat misleading. These should be `{input|expected}_image_tensor`.
2. Is there a reason to keep both methods separate? IIUC, they are always requested together so it might makes sense to merge them into one method to have everything in a single place.
3. You could also accept a `p` parameter here and switch the expected tensor based on it. This way the test only needs to handle the type rather than the values.

Reply:
Just a doubt about 1.
Do you refer to the two methods or to the variable names in tests?

For example:
```
input, expected = self.input_expected_image_tensor(p, dtype=torch.uint8)
transform = transforms.RandomHorizontalFlip(p=p)

actual = transform(to_pil_image(input))

assert_equal(expected, pil_to_tensor(actual))
```

`input` and `expected` are decorated in this case, so maybe it might be more correct using `input_image_tensor` and `expected_image_tensor`. On the other hand, they are very long and verbose variable names, and the method name is quite explicit now.